lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,69 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+
+      DEFAULT_STEP = 1 
+      APP_MIGRATION_DIRECOTRY = "apps/web/db/migrations".freeze
+      MIGRATION_DIRECTORY = "db/migrations".freeze
```

Comment:
@vinioliveira I don't think Lotus::Model::Migration should be aware of the directories defined by Lotus (lotusrb). Firstly, Lotus::Model is designed to be framework agnostic. Secondly, I don't think Lotus::Model is the right place to hold this value.